python/ray/tune/tests/test_tune_restore.py

```diff
@@ -212,13 +212,19 @@ def cost(space, reporter):
             space,
             metric="loss",
             mode="min",
+            analysis=analysis,
             utility_kwargs={
                 "kind": "ucb",
                 "kappa": 2.5,
                 "xi": 0.0
             })
         return search_alg, cost
 
+    def testWarmStart(self):
+        super().testWarmStart()
+        analysis = self.run_exp_3()
+        self.set_basic_conf(analysis)
```

Comment:
```suggestion
    def testBootStrapAnalysis(self):
        analysis = self.run_exp_3()
        search_alg3, cost = self.set_basic_conf(analysis)
        search_alg3 = ConcurrencyLimiter(search_alg3, 1)
        tune.run(
            cost, num_samples=10, search_alg=search_alg3, verbose=0)
```

This should work right?